packages/flutter_tools/templates/app/lib/main.dart.tmpl

```diff
@@ -113,14 +115,16 @@ class _MyHomePageState extends State<MyHomePage> {
       floatingActionButton: FloatingActionButton(
         onPressed: _incrementCounter,
         tooltip: 'Increment',
-        child: Icon(Icons.add),
+        child: const Icon(Icons.add),
       ), // This trailing comma makes auto-formatting nicer for build methods.
     );
   }
 }
 {{/withPluginHook}}
 {{#withPluginHook}}
 class MyApp extends StatefulWidget {
+  const MyApp({Key? key}) : super(key: key);
+
   @override
   _MyAppState createState() => _MyAppState();
```

Comment:
Does this conflict with the private return types lint? Or is that not in the recommended set

Reply:
That's not in the recommended set.